testsuite/gnat2goto/tests/loops/test.out

```diff
@@ -1,2 +1,4 @@
+Standard_Error from gnat2goto loops:
+loops.adb:9:04: warning: variable "i1" is never read and never assigned
```

Comment:
It is rather annoying to have to include warnings in the expected output. I'd prefer leaving it out, but I don't want to change the test script _again_ in this PR

Reply:
There's an argument to say that warnings are part of the user-visible output, so any *change* in warnings emitted by the tools should be noticed.